src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -121,6 +124,14 @@ public CachedSchemaRegistryClient(IEnumerable<KeyValuePair<string, string>> conf
                 throw new ArgumentException($"Invalid value '{basicAuthSource}' specified for property '{SchemaRegistryConfig.PropertyNames.SchemaRegistryBasicAuthCredentialsSource}'");
             }
 
+            var keySubjNameStrategy = Convert.ToString(config.FirstOrDefault(prop => prop.Key.ToLower() == SchemaRegistryConfig.PropertyNames.SchemaRegistryKeySubjectNameStrategy).Value) ?? "";
+            var valueSubjNameStrategy = Convert.ToString(config.FirstOrDefault(prop => prop.Key.ToLower() == SchemaRegistryConfig.PropertyNames.SchemaRegistryValueSubjectNameStrategy).Value) ?? "";
+
+            if(Enum.TryParse<SubjectNameStrategy>(keySubjNameStrategy, out var strategy))
```

Comment:
keep formatting consistent with other parts of the codebase (space after if, always use `{}` even when not necessary).

Reply:
Hi, I corrected it and pushed.